recipes/lzham/all/conandata.yml

```diff
@@ -0,0 +1,29 @@
+sources:
+  "1.0.0":
+    sha256: "4f4f874706763b3a6e3d6dfff666a1e850ca1d92fd9240b2a14365c5864a0057"
+    url: "https://github.com/richgel999/lzham_codec/archive/refs/tags/v1_0_stable1.tar.gz"
+patches:
+  "1.0.0":
+    - patch_file: "patches/commits-1.0.0.patch"
+      patch_description: 'Updates code to latest commit for the repo
```

Comment:
The first option is ideal and I've made an [issue](https://github.com/richgel999/lzham_codec/issues/32) for. Unfortunately the maintainers are pretty unresponsive so I doubt it will come to pass haha. I think the second option doesn't apply as this code is already part of the upstream, though maybe I misunderstood. I made [pull requests](https://github.com/richgel999/lzham_codec/pulls) for my other patches where they did not already exist.

I think it is likely that I will go for the "unofficial releases" option

Reply:
Also, is it acceptable if I supported *only* the unofficial release? I ask because it contains many portability fixes, and the official would basically have to be patched to include the fixes anyway if I include it

Reply:
I think it makes sense in this case given how much differences there are